flow/Platform.actor.cpp

```diff
@@ -2324,7 +2318,7 @@ void atomicReplace(std::filesystem::path const& path, std::string const& content
 		INJECT_FAULT(io_error, "atomicReplace"); // atomic rename failed
 
 		std::string tempfilename =
```

Comment:
which version is set, if we support C++20 upwards this is a good case for std::format.  Since we're creating at least 2 tmp strings.

Reply:
I believe this is for C++17 but if maintainers can verify this that can be helpful, `std::format` would be the better way to go about this